tests/Nox.Types.Tests/Types/StreetAddress/StreetAddressTests.cs

```diff
@@ -3,9 +3,54 @@ namespace Nox.Types.Tests.Types;
 
 public class StreetAddressTests
 {
+    [Theory]
+    [InlineData(5, "Line1", "", "Locality", "11111", "UA", "Line1, Locality, UA 11111")]
+    [InlineData(5, "", "Line2", "Locality", "11111", "UA", "Line2, Locality, UA 11111")]
+    [InlineData(5, "", "", "Locality", "11111", "UA", "Locality, UA 11111")]
+    [InlineData(5, "Line1", "Line2", "Locality", "11111", "UA", "Line1 Line2, Locality, UA 11111")]
+    public void Nox_StreetAddress_AddressString_ReturnsValidAddress(
```

Comment:
you can remove the Nox prefix from all tests, its redundant and we are doing this now
thks

We are following this convention for naming tests https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#best-practices => Method,Scenario,Expected
```suggestion
    public void ToString_StreetAddress_ReturnsValidAddress(
```